src/Guzzle6HttpAdapter.php

```diff
@@ -8,15 +8,15 @@
  * For the full copyright and license information, please read the LICENSE
  * file that was distributed with this source code.
  */
-
 namespace Http\Adapter;
 
 use GuzzleHttp\Client;
 use GuzzleHttp\ClientInterface;
-use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Exception as Guzzle;
```

Comment:
Nope, it is three. Well, in this case namespace importing might make sense, but alias it at least to GuzzleExceptions namespace.